diff -ur dcmtk2/dcmdata/libsrc/vrscanl.c dcmtk1/dcmdata/libsrc/vrscanl.c
--- dcmtk2/dcmdata/libsrc/vrscanl.c	2010-10-14 13:07:00.000000000 +0000
+++ dcmtk1/dcmdata/libsrc/vrscanl.c	2011-10-31 17:41:34.398884922 +0000
@@ -5603,7 +5603,6 @@
  * %option nounistd
  */
 #line 120 "vrscanl.l"
-static int isatty(int fd) { return 0; }
 #line 5605 "vrscanl.c"
 
 #define INITIAL 0
diff -ur dcmtk2/dcmwlm/data/Makefile.in dcmtk1/dcmwlm/data/Makefile.in
--- dcmtk2/dcmwlm/data/Makefile.in	2010-12-06 10:20:01.000000000 +0000
+++ dcmtk1/dcmwlm/data/Makefile.in	2011-10-31 17:36:44.805339017 +0000
@@ -20,7 +20,7 @@
 	$(configdir)/mkinstalldirs $(DESTDIR)$(datadir)
 	for folder in $(folders); do \
 		$(configdir)/mkinstalldirs $(DESTDIR)$(datadir)/$$folder ;\
-		find $$folder -maxdepth 1 -type f -exec $(INSTALL_DATA) {} $(DESTDIR)$(datadir)/$$folder \; ;\
+		find $$folder -prune -type f -exec $(INSTALL_DATA) {} $(DESTDIR)$(datadir)/$$folder \; ;\
 	done
 
 clean:
